app/assets/javascripts/discourse/app/templates/modal/edit-user-directory-columns.hbs

```diff
@@ -0,0 +1,48 @@
+{{#d-modal-body title="directory.edit_columns.title"}}
+  {{#if loading}}
+    {{loading-spinner size="large"}}
+  {{else}}
+    <div class="edit-directory-columns-container">
+      {{#each columns as |column|}}
+        <div class="edit-directory-column">
+          <div class="left-content">
+            <label class="column-name">
```

Comment:
This is repeated almost line for line in an earlier component. Maybe make a component to DRY it up?

Reply:
I think you're talking about this: 
https://github.com/discourse/discourse/blob/300f4c910f9495d6fe1664f95239b376c7eee42e/app/assets/javascripts/discourse/app/templates/modal/edit-user-directory-columns.hbs#L11-L14

Which looks visually the same as this 
https://github.com/discourse/discourse/blob/300f4c910f9495d6fe1664f95239b376c7eee42e/app/assets/javascripts/discourse/app/templates/components/directory-item.hbs#L4-L8

But use all different helpers

Reply:
My bad! Sorry.